Code/Framework/AzFramework/AzFramework/Physics/Configuration/RigidBodyConfiguration.h

```diff
@@ -57,6 +57,7 @@ namespace AzPhysics
         bool m_ccdEnabled = false; //!< Whether continuous collision detection is enabled.
         float m_ccdMinAdvanceCoefficient = 0.15f; //!< Coefficient affecting how granularly time is subdivided in CCD.
         bool m_ccdFrictionEnabled = false; //!< Whether friction is applied when resolving CCD collisions.
+        bool m_configButton = false; // required for button positioning in edit context, otherwise unused
```

Comment:
I actually fixed the `UIElement` ordering issue earlier this year: https://github.com/o3de/o3de/pull/9569

So we don't need to do the workaround anymore by having an unused data element, so this can be removed and below in the EditContext you can just do:

`->UIElement(AZ::Edit::UIHandlers::Button, ...`

Reply:
I tried that first and just again now and the button doesn't appear in the right place: 
![image](https://user-images.githubusercontent.com/105638312/205976111-1e7e485f-139f-4333-aaa4-fa3e7d3cdaf0.png)

Reply:
Any suggestions @cgalvan, it seems this may have regressed? Can we keep the data element for now?

Reply:
Weird, I see the same issue locally for this component, it works fine in other components in sub-groups so I'm not sure what the issue is, I'll remove my change request. It's probably not worth investigating further with the efforts on the DPE.